src/analysis/retail/hunter/marksmanship/modules/Abilities.tsx

```diff
@@ -21,7 +22,12 @@ class Abilities extends CoreAbilities {
         spell: TALENTS.AIMED_SHOT_TALENT.id,
         enabled: this.selectedCombatant.hasTalent(TALENTS.AIMED_SHOT_TALENT),
         category: SPELL_CATEGORY.ROTATIONAL,
-        cooldown: (haste: number) => 12 / (1 + haste),
+        cooldown: (haste: number) => {
+          if (this.selectedCombatant.hasTalent(TALENTS.TACTICAL_RELOAD_TALENT)) {
+            return 12 / (1 + haste + TACTICAL_RELOAD_CDR_REDUCTION);
+          }
+          return 12 / (1 + haste);
+        },
```

Comment:
I don't think this is correct, because talents like this usually stack multiplicatively with haste, not additively. Also "CD reduced by 10%" in blizz speak means "multiply by 0.9", not "divide by 1.1". The correct formula should be `12 * (1 - TACTICAL_RELOAD_CDR_REDUCTION) / (1 + haste)`

Reply:
appreciate the correct formula. Was totally a guess in the dark on my end